repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -135,13 +139,26 @@ def process_events(events):
         api.produce(RepositoriesSetupTasks(to_enable=list(to_enable_repos)))
 
 
+def verify_pes_data(path):
+    """Verify if PES data is available if not inhibit upgrade"""
+    if path is None or not os.path.isfile(path):
+        title = 'PES data is invalid (/etc/leapp/files/pes-data.json)'
```

Comment:
```suggestion
        title = 'Missing PES data ({})'.format(path)
```

Invalid doesn't seem right if you only check existence of the file